llvm/include/llvm/ADT/GenericUniformityImpl.h

```diff
@@ -1210,6 +1240,13 @@ void GenericUniformityAnalysisImpl<ContextT>::print(raw_ostream &OS) const {
   }
 }
 
+template <typename ContextT>
+iterator_range<typename GenericUniformityInfo<ContextT>::UOCWDE *>
```

Comment:
Just say ``auto`` as the return type here? Or if this needs to be exposed in an outer header file, then name a new type such as ``temporal_divergence_range``?